assignments/assignment-8/assignment-8.2.html

```diff
@@ -0,0 +1,136 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Assignment 8.2</title>
+    </head>
+    <body>
+        <form>
+            <h2>Quiz</h2>
+
+            <label for="fullNameInput">Full Name</label>
+            <input
+                type="text"
+                placeholder="Full Name"
+                id="fullNameInput"
+                name="fullName"
+                required
+            /><br />
+            <br />
+            <label for="emailInput">Email</label>
+            <input
+                type="email"
+                placeholder="Email"
+                id="emailInput"
+                name="email"
+                required
+            />
+
+            <h3>Confidence level of answering</h3>
+
+            <label for="confidenceLevelInput">Level between 1 and 10</label>
+            <input
+                type="range"
+                id="confidenceLevelInput"
+                name="confidenceLevel"
+                min="1"
+                max="10"
+            />
+
+            <label for="ageInput">Age</label>
+            <input
+                type="number"
+                id="ageInput"
+                name="age"
+                min="12"
+                max="65"
+                required
+            /><br /><br />
+
+            <h2>Preferred Topics</h2>
+            <input
+                type="checkbox"
+                id="preferredHistoryChexkbox"
+                name="history"
```

Comment:
isHistoryChecked better for name attr

Reply:
ok sure

Reply:
name=“isHistoryFavoriteQuizCategory” is my option. like “isSportsPreferredTopic”